source/get-started.txt

```diff
@@ -209,18 +209,18 @@ your operating system:
         
       .. code-block:: cmake
        
-         find_package(mongoc-1.0 CONFIG REQUIRED)
-         target_link_libraries(my-application
-             PRIVATE $<IF:$<TARGET_EXISTS:mongo::mongoc_shared>,mongo::mongoc_shared,mongo::mongoc_static>)
+         find_package(mongoc CONFIG REQUIRED)
+         target_link_libraries(my-application PRIVATE mongoc::mongoc)
         
       .. note::
         
-         You can use the ``$<IF:$<TARGET_EXISTS...>:...>`` generator
-         expression in the preceding example to switch the link type of
-         ``libmongoc`` based on the form that is available from the
-         ``find_package()`` command. ``libmongoc`` supports building with
-         both *dynamic* and *static* library types, but vcpkg
-         installs only one type at a time.
+         In the preceding example, the ``mongoc::mongoc`` target
+         points to either the static library or the shared library.
+         The library type depends on which one is available and
+         whichever type the user specifies in the ``MONGOC_DEFAULT_IMPORTED_LIBRARY_TYPE``
+         configuration setting. If you don't set this value and
+         both library types are available, ``libmongoc`` builds with
+         the static library.
```

Comment:
Wording tweak:

```suggestion
         both library types are available, ``mongoc::mongoc`` uses
         the static library.
```

Would also add a note about the similar `bson::bson` and `BSON_DEFAULT_IMPORTED_LIBRARY_TYPE` options.

Reply:
Fixed! I'll add the bson information to the [What's New](https://jira.mongodb.org/browse/DOCSP-48738)